### PR TITLE
test: Code coverage - Token service - 100% branches

### PR DESCRIPTION
**Pull request by Dimple16**
click up - https://app.clickup.com/t/85zrpu3mx

Screenshot 2023-03-14 at 1 03 12 PM

### CODE REVIEW

File: src/app/core/services/token.service.spec.ts [hunk 1]
```diff
@@ -1,16 +1,107 @@
...
-    expect(service).toBeTruthy();
+    expect(tokenService).toBeTruthy();
+  });
+
+  it('getAccessToken(): should get access token from secure storage', (done) => {
+    secureStorageService.get.and.returnValue(Promise.resolve('token'));
+    tokenService.getAccessToken().then((token) => {
+      expect(token).toEqual('token');
+      expect(secureStorageService.get).toHaveBeenCalledOnceWith('X-AUTH-TOKEN');
+      done();
```
Comment (rvab) [inline src/app/core/services/token.service.spec.ts:37]: iirc, we should use `tick` for promises. Could you check this once?

File: src/app/core/services/token.service.spec.ts [hunk 2]
```diff
@@ -1,16 +1,107 @@
...
+    });
+  });
+
+  it('setAccessToken(): should set access token to secure storage', (done) => {
+    userEventService.setToken.and.returnValue();
```
Comment (rvab) [inline src/app/core/services/token.service.spec.ts:42]: is this intentional? returning empty value?

**Review by rvab — CHANGES_REQUESTED** [review]
One important comment

File: src/app/core/services/token.service.spec.ts (hunk 1 above)
Comment (Dimple16) [reply src/app/core/services/token.service.spec.ts]: fixed

File: src/app/core/services/token.service.spec.ts (hunk 2 above)
Comment (Dimple16) [reply src/app/core/services/token.service.spec.ts]: missed `null`, fixed

**Review by rvab — APPROVED** [review]

**Pull request merged**